statik/.gitignore

```diff
@@ -1 +1 @@
-statik.go
+/statik.go
```

Comment:
will this ignore any file called statik.go? We should probably make it ignore just the one that gets generated, right?

Reply:
Oh I see I'm mistaken and that's what it does.